requirements.txt

```
@@ -1,2 +1,3 @@
-requests
-pywal
\ No newline at end of file
+requests==2.24.0
```

Can you unpin these versions? or at least make it so it will pull in any new minor versions?
```suggestion
requests==2.*
```

---

Done